app/Resources/translations/default/AdminAdvparametersFeature.xlf

```diff
@@ -106,6 +115,20 @@
       </trans-unit>
     </body>
   </file>
+  <file original="classes/webservice/WebserviceKey.php" source-language="en-US" target-language="en" datatype="plaintext">
+    <body>
+      <trans-unit id="9b4cd8a4fb5c6e01d647bfb05407f909">
+        <source>Webservice key created %s</source>
```

Comment:
The %s is replaced with the key that has just been created, so I guess it could be:
`Webservice key created: %s`
or
`Webservice key %s created`

This wording is used in the logs only

Reply:
Thanks! Both suit me fine but if we use the first one, maybe we need to add the ":" so that it is clearer. :)

Reply:
@jolelievre